images/win/Windows2016-Azure.json

```diff
@@ -296,6 +296,12 @@
                 "{{ template_dir }}/scripts/Installers/Install-Bazel.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "scripts":[
+                "{{ template_dir }}/scripts/Installers/Validate-SQLExpress.ps1"
```

Comment:
Probably, we should install tool before validation?

Reply:
You are absolutely right, thank you.